src/providers/vod/vod-provider.ts

```diff
@@ -35,16 +37,27 @@ export class VodProvider extends Provider {
       this._eventManager.listen(this._player, this._player.Event.TEXT_TRACK_CHANGED, this._handleLanguageChange);
     }
     if (this._isPreventSeekActive) {
-      this._eventManager.listen(this._player, this._player.Event.TIME_UPDATE, () => this._onTimeUpdate());
+      this._eventManager.listen(this._player, this._player.Event.TIME_UPDATE, this._onTimeUpdate);
```

Comment:
if you remove the wrapper function the value of this can be wrong. any specific reason to remove it ?

Reply:
nothing special- just aligned it with the other callbacks there and make sure it works this way too.

Reply:
@semarche-kaltura we tried that, there were issues with it